Files/Dialogs/DecompressArchiveDialog.xaml

```diff
@@ -0,0 +1,50 @@
+﻿<ContentDialog
+    x:Class="Files.Dialogs.DecompressArchiveDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:helpers="using:Files.Helpers"
+    xmlns:local="using:Files.Dialogs"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Title="Select a destination and extract files"
+    CornerRadius="{StaticResource OverlayCornerRadius}"
+    DefaultButton="Primary"
+    PrimaryButtonText="Extract"
+    RequestedTheme="{x:Bind helpers:ThemeHelper.RootTheme}"
+    SecondaryButtonText="Cancel"
+    Style="{StaticResource DefaultContentDialogStyle}"
+    mc:Ignorable="d">
+
+    <Grid
+        MinWidth="400"
+        Margin="0,16,0,0"
```

Comment:
Doesn't the ContentDialog already have its own margin?

Reply:
The title and content was too close to eachother